executor/cook/server.py

```diff
@@ -18,12 +18,20 @@ def do_GET(self):
         """
         GET /health
           => returns 200
+        GET /task/:task_id
+          => returns 200 and task data if task_id is valid
+          => returns 404 if task_id is unknown or invalid
         """
         if self.path is '/health':
-            self.send_response(200)
-            self.send_header('Content-type', 'application/json')
-            self.end_headers()
-            self.wfile.write(json.dumps('{"response": "okay"}'))
+            self.send_json_response(200, {'response': 'okay'})
+        elif re.match('\/task\/[^/]+', self.path):
```

Comment:
I think I might be missing something.

There should only exist 1 task data for the executor right? Or is this per command?